### ISSUE TITLE
[Community] Help me tag your favorite RPG!

### ISSUE THREAD

**Issue by isaaclyman** [isaaclyman/ttrpg-guide#3]
It would be useful to add tags to each RPG to give a sense of the way it feels to play.

Please comment with an RPG you've played (especially if you've GM'd it) and the tags you feel are *most* representative of gameplay.

Tag examples:

- action
- bite-sized
- cinematic
- deadly
- drama
- fast-paced
- gritty
- heist
- humor
- pulpy
- realism
- tactical

**Comment by isaaclyman** [comment]
**Dungeons and Dragons 5E**

Suggested tags:

- action
- tactical

**Comment by Eleksar387** [comment]
I suggest solo. There is a growning solo community [r/Solo_Roleplaying](https://www.reddit.com/r/Solo_Roleplaying/).

**Comment by isaaclyman** [comment]
@Eleksar387 Thanks for the suggestion. Are you aware of any solo RPGs that meet the [Criteria for inclusion](https://github.com/isaaclyman/ttrpg-guide?tab=readme-ov-file#criteria)?

**Comment by Eleksar387** [comment]
Several . Not all of them are pure solo but can be played solo.
https://www.reddit.com/r/FourAgainstDarkness/
https://www.reddit.com/r/MorkBorg/
https://www.reddit.com/r/ForbiddenLands/

**Comment by isaaclyman** [comment]
Perfect. Mork Borg is already listed, so I'll add the `solo` tag to it and put the other two on the [wishlist](https://github.com/isaaclyman/ttrpg-guide?tab=readme-ov-file#to-add).

**Comment by Eleksar387** [comment]
I am willing to contribute more. It was actually just a "test" as I was unsure if this is still active 😁

**Comment by isaaclyman** [comment]
My focus has been on other projects for the last couple months (e.g. I just changed jobs) but I still intend to maintain this project.

**Comment by Eleksar387** [comment]
> EDIT: On second look, Mork Borg doesn't ship with solo rules. You can use an oracle, but that goes for every RPG, and therefore isn't a distinguishing characteristic. Ditto for Forbidden Lands. Since Four Against Darkness is built to purpose as a solo game, the `solo` tag would only be appropriate for that one.

True. Mork Borg has an unoffical extension https://1d105.itch.io/solitary-defilement

That's actually true for a lot of RPGs: The One Ring, Dolmenwood, Across A Thousand Dead Worlds,.

**Comment by isaaclyman** [comment]
Right. If there's going to be a `solo` tag, I'd like it to mean "the publisher had solo roleplayers in mind."
